doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,161 @@
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Fop State Machine (fom)                                 | fom is a state machine that represents current state of the fop's [r.fop] ST execution on a node. fom is associated with the particular fop and implicitly includes this fop as part of its state                                                                                                                                                                                                                                                                                                                                                                                      |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Group                                                   | The whole space is equally divided into groups with fixed size. That means every group has the same amount of blocks. When allocating spaces, groups are iterated to search for the best candidate. Group is locked during this step. Having multiple groups in a container can reduce the lock contention.                                                                                                                                                                                                                                                                            |
```

Comment:
Maybe, we can say: Container Allocation Group.

Just a single word, group, is very vague.